tools/vhost-user-video/virtio_video_udmabuf.c

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Virtio Video Device
+ *
+ * Copyright Red Hat, Inc. 2023
+ *
+ * Authors:
+ *     Albert Esteve <aesteve@redhat.com>
+ *
+ * This work is licensed under the terms of the GNU GPL, version 2 or later.
+ * See the COPYING file in the top-level directory.
+ */
+
+#include "qemu/osdep.h"
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+
+#include "vuvideo.h"
+
+struct udmabuf_create {
+    uint32_t memfd;
+    uint32_t flags;
+    uint64_t offset;
+    uint64_t size;
+};
+
+#define UDMABUF_CREATE _IOW('u', 0x42, struct udmabuf_create)
```

Comment:
I believe you have this define already in the `linux/udmabuf.h` header, so just include that.